### PR TITLE
task-3868 themed Editor Picks block

### PR DESCRIPTION
**Pull request by TerryKolodii**

### CODE REVIEW

File: web/themes/custom/news/dist/main.css [hunk 1]
```diff
@@ -420,10 +420,10 @@ a, .link {
       color: #000000; }
```
Comment (henzzapp) [inline web/themes/custom/news/dist/main.css]: @TerryKolodii 
We don't commit dist folder contents

it should be in .gitignore

Comment (TerryKolodii) [reply web/themes/custom/news/dist/main.css]: fixed

Comment (henzzapp) [reply web/themes/custom/news/dist/main.css]: @TerryKolodii you need to delete the file, then commit it.

Afterwards add it to gitignore.

Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we'
ll see

Comment (TerryKolodii) [reply web/themes/custom/news/dist/main.css:421]: > Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we' ll see
do you mean I need to do that with my next PR? Not in this one?

Comment (TerryKolodii) [reply web/themes/custom/news/dist/main.css]: > Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we'
> ll see

Do you mean I need to do that with  my next PR? Not in this one?

**Pull request merged**